bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/ArchiveWorkItem.java

```diff
@@ -255,9 +243,14 @@ private String subjectPrefix() {
         var path = scratchPath.resolve("mlbridge");
         var archiveRepo = materializeArchive(path);
         var mboxBasePath = path.resolve(bot.codeRepo().name());
-        var mbox = MailingListServerFactory.createMboxFileServer(mboxBasePath);
-        var reviewArchiveList = mbox.getListReader(pr.id());
-        var sentMails = parseArchive(reviewArchiveList);
+
+        var sentMails = new ArrayList<Email>();
+        try {
+            var archiveContents = Files.readString(mboxBasePath.resolve(pr.id() + ".mbox"), StandardCharsets.UTF_8);
+            sentMails.addAll(Mbox.splitMbox(archiveContents, bot.emailAddress()));
+        } catch (IOException ignored) {
```

Comment:
Shouldn't we at least log this?

Reply:
This will happen every time a new PR is processed (since there won't be any archive) - I can't think of any actual error that could cause a problem here (problems with the folder contents should have been discovered when cloning it from git)